app/controllers/lookups_controller.rb

```diff
@@ -1,7 +1,12 @@
 class LookupsController < ApplicationController
   def by_base_path
+    # return content_ids for content that is visible on the live site
+    # withdrawn items are still visible
+    states = %w(published unpublished)
```

Comment:
@MatMoore AIUI clients of this endpoint are currently the checker tool and content-tagger. Neither of these needs to distinguish withdrawn content using the results of this endpoint (although they might need to know in other circumstances). In particular, the checker tool only uses this output to link up the iterative imports it has done from Rummager and Publishing API.